#use wml::debian::translation-check translation="1.1" maintainer="Pierre Machard"
<define-tag description>Fichiers temporaires non sécurisés</define-tag>
<define-tag moreinfo>
<p>Zack Weinberg a découvert une utilisation non sécurisée de fichiers
temporaires dans <i>os._execvpe</i> de <i>os.py</i>. Il utilise un nom
qu'on peut déterminer à l'avance qui peut permettre l'exécution d'un
code arbitraire.</p>

<p>Ce problème a été corrigé dans plusieurs versions de Python&nbsp;:
pour l'actuelle distribution stable (<i>Woody</i>), ça a été corrigé
dans la version&nbsp;1.5.2-23.1 de  Python&nbsp;1.5, dans la 
version&nbsp;2.1.3-3.1 de Python&nbsp;2.1 et dans la version&nbsp;1.5.2-23.1
de Python&nbsp;2.2. Pour l'ancienne distribution stable (<i>Potato</i>)
cela a été corrigé dans la version&nbsp;1.5.2-10potato12 pour
Python&nbsp;1.5. Pour la distribution <i>unstable</i> (<i>Sid</i>) ceci
a été corrigé dans la version&nbsp;1.5.2-24 de Python&nbsp;1.5, dans
la version&nbsp;2.1.3-6a de Python&nbsp;2.1 et dans la version&nbsp;2.2.1-8 de
Python&nbsp;2.2. Python&nbsp;2.3 n'est pas affecté par cette faille.</p>

<p>Nous vous recommandons de mettre à jour vos paquets Python immédiatement.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2002/dsa-159.data"
